src/Octoshift/Services/MigrateRepoErrorHandler.cs

```diff
@@ -0,0 +1,18 @@
+using System.Diagnostics.Contracts;
+
+namespace OctoshiftCLI.Services;
+
+public static class MigrateRepoErrorHandler
+{
+    public static OctoshiftCliException DecorateInsufficientPermissionsException(OctoshiftCliException ex, string orgName)
+    {
+        Contract.Requires(ex != null);
```

Comment:
I haven't seen this before in C#, but the formatter seemed to want me to add it. Very open to alternative approaches here!